fhireval/data/CMG-Extras.json

```diff
@@ -0,0 +1,226 @@
+{
+  "groupname": "CMG",
+  "datatype": "Data",
+  "resources": {
+    "Group": [
+      {
+        "resourceType": "Group",
+        "identifier": [
+          {
+            "system": "urn:ncpi-test:unique-string",
+            "value": "patient-export-test-01"
+          }
+        ],
+        "type": "person",
+        "actual": true,
+        "code": {
+          "text": "Exportable Folks"
+        },
+        "name": "Bulk Export Test Group",
+        "member": [
+          {
+            "entity": {
+              "reference": "Patient/CMG-Patient-1"
+            }
+          },
+          {
+            "entity": {
+              "reference": "Patient/CMG-Patient-3"
+            }
+          },
+          {
+            "entity": {
+              "reference": "Patient/CMG-Patient-6"
+            }
+          }
+        ]
+      },
+      {
+        "resourceType": "Group",
+        "identifier": [
+          {
+            "system": "http://someveterinarianclinic.org/fhir/NamingSystem/herds",
+            "value": "12345"
+          }
+        ],
+        "type": "animal",
+        "actual": true,
+        "code": {
+          "text": "Horse"
+        },
+        "name": "John's herd",
+        "quantity": 25,
```

Comment:
It has `"quantity"`, but `"member"`. Is this intended?

Reply:
Hm, not entirely sure I understand what you are saying, however, it may be that you are seeing members and quantity in the same group, but that's not true. There are two groups: The first group of persons has 3 members and the other is a group of horses (that I stole from some HL7 example for some early tests). I could probably revisit that initial set of tests and refactor to get rid of that group of horses, but for now, it seems reasonable to leave them both in.